tensorflow_probability/python/distributions/pert.py

```diff
@@ -0,0 +1,164 @@
+"""The PERT distribution class"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow.compat.v2 as tf
+from tensorflow_probability.python.distributions import Beta
+from tensorflow_probability.python.bijectors import AffineScalar
+from tensorflow_probability.python.distributions import transformed_distribution
+from tensorflow_probability.python.internal import assert_util
+from tensorflow_probability.python.internal import dtype_util
+from tensorflow_probability.python.internal import prefer_static
+from tensorflow_probability.python.internal import tensor_util # pylint: disable=g-direct-tensorflow-import
+
+__all__ = [
+    'PERT',
+]
+
+class PERT(transformed_distribution.TransformedDistribution):
+  """Modified PERT distribution:
```

Comment:
the first line should be a one-liner "sentence" e.g. "Modified PERT distribution for modeling expert predictions."
then an empty line.

can you add a "Mathematical details" section a la https://github.com/tensorflow/probability/blob/master/tensorflow_probability/python/distributions/gamma_gamma.py#L51
and optionally an example or two a la
https://github.com/tensorflow/probability/blob/master/tensorflow_probability/python/distributions/dirichlet.py#L103 ?

Reply:
Added a simple mathematical overview and couple of examples demonstrating broadcasting behaviors.

https://github.com/tensorflow/probability/blob/33ea65ee4be4ed05247357f7d5b374a38d3b3f58/tensorflow_probability/python/distributions/pert.py#L21